Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM quay.io/buildah/stable:v1.14.3
```

Comment:
Is there enough overlap to justify using the `che-sidecar-kubernetes-tooling` image as a base?